pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-kafka/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: layout.pug
+navigationTitle: KUDO Kafka
+render: mustache
+title: KUDO Kafka
+menuWeight: 3
+excerpt: Day 2 Operations of KUDO Kafka
+beta: false
+---
+
+<!-- markdownlint-disable MD018 -->
+## KUDO Kafka
+
+Kommander Catalog adds integration for [KUDO Kafka Operator](https://github.com/mesosphere/kudo-kafka-operator/), which simplifies day 2 operations of [Apache Kafka](https://kafka.apache.org/). 
+
+#include /dkp/kommander/1.3/include/kudo-intro.tmpl
+
+It is **strongly recommended** to view the [KUDO Kafka Documentation](https://kudo.dev/docs/runbooks/kafka/install.html) which covers the KUDO Kafka Operator in-depth. This document covers the integration aspects of KUDO Kafka Operator with D2iQ Kommander.
+
+### Kommander Catalog
+
+KUDO Kafka is located in the Kommander Catalog. To access the catalog: 
+#include /dkp/kommander/1.3/include/kommander-catalog-drilldown.tmpl
+
+### Zookeeper
+
+Apache Kafka has a dependency on [Apache Zookeeper](https://zookeeper.apache.org/). The Kommander Catalog includes [KUDO Zookeeper](/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-zookeeper/) to be used in conjunction with KUDO Kafka.
+
+The `ZOOKEEPER_URI` parameter configures which Zookeeper cluster to use.
+- KUDO Zookeeper uses the following format: `<id>-cs.<namespace>:<client_port>`
+- With `zookeeper` launched in the namespace `test-project-zc6tc` we have this as the `ZOOKEEPER_URI`: `zookeeper-cs.test-project-zc6tc:2181`
+
+The `ZOOKEEPER_PATH` parameter configures the zNode path within the Zookeeper Cluster to use.
+- When unspecified the zNode defaults to the id of the current Kafka instance being launched.
+- When specified a mandatory leading slash is needed i.e `/kafka`
+
+### Installation
+
+From the [Project Catalog](/dkp/kommander/1.3/projects/platform-services/) select the desired version of Kafka and click Deploy.
+
+Below is an example of what should appear in the Kommander UI.  The dialog is populated with sensible defaults:
```

Comment:
I just tried to install Kafka 2.5.1-1.  The 'sensible default' :-) for the zookeeper URI is this string:

`zookeeper-zookeeper-0.zookeeper-hs:2181,zookeeper-zookeeper-1.zookeeper-hs:2181,zookeeper-zookeeper-2.zookeeper-hs:2181`

This is what led to a lot of my confusion, in particular the reference to the zookeeper-hs namespace...  Probably not a docs issue, tho.  I can confirm that using <zk-install-name>-cs.<namespace>:2181 works here.

Reply:
@cneth yeah, we're fixing that in upstream KUDO :)